include/zcbor_decode.h

```diff
@@ -73,7 +73,6 @@ bool zcbor_uint_decode(zcbor_state_t *state, void *result, size_t result_size);
 bool zcbor_bstr_decode(zcbor_state_t *state, struct zcbor_string *result); /* bstr */
 bool zcbor_tstr_decode(zcbor_state_t *state, struct zcbor_string *result); /* tstr */
 bool zcbor_tag_decode(zcbor_state_t *state, uint32_t *result);  /* CBOR tag */
-bool zcbor_simple_decode(zcbor_state_t *state, uint8_t *result); /* CBOR simple value */
```

Comment:
Should we document this as its an API breaking change? Also we would like to have this before 1.0 I guess to avoid more breakage.

Reply:
For 0.7.0 -> 0.8.0 I created some migration docs for Zephyr. I've thought about having it here instead, and this is a reason to do it. I created a task for it and will add it before the next release, whether it's 0.9.0 or 1.0.0